packages/react/src/components/layout/Drawer/index.tsx

```diff
@@ -171,10 +172,19 @@ const DrawerContent = ({
   );
 };
 
-const Drawer = ({ children, ...sideProps }: DrawerProps): React.ReactElement => {
-  const $root = React.useMemo(() => document.querySelector("#ll-side-root"), []);
-  if ($root === null) throw new Error("side root cannot be found");
-  return ReactDOM.createPortal(<DrawerContent {...sideProps}>{children}</DrawerContent>, $root);
+const Drawer = ({ children, menuPortalTarget, ...sideProps }: DrawerProps): React.ReactElement => {
+  const $root = React.useMemo(
```

Comment:
why the $variable though? :thinking: im used of this notation for observables

Reply:
you tell me :D

https://github.com/LedgerHQ/ui/blame/4403598b6aa4c06c450f2e14cee9eac32190e3d6/packages/react/src/components/layout/Drawer/index.tsx#L175

Reply:
I'm nostalgic, this notation smells jquery